core/chain-configs/src/lib.rs

```diff
@@ -7,4 +7,4 @@ pub use genesis_config::{
 };
 
 /// Current latest version of the protocol
-pub const PROTOCOL_VERSION: u32 = 10;
+pub const PROTOCOL_VERSION: u32 = 11;
```

Comment:
@bowenwang1996 we should make sure we don't collide with #2504. Just a heads up.

Reply:
We will not because the state migration check on CI will fail.